docs/content/preview/drivers-orms/orms/php/ysql-laravel.md

```diff
@@ -0,0 +1,284 @@
+---
+title: PHP ORM example application
+headerTitle: PHP ORM example application
+linkTitle: PHP
+description: PHP ORM example application.
+menu:
+  preview:
+    identifier: php-orm
+    parent: orm-tutorials
+    weight: 660
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../ysql-gorm/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      Laravel ORM
+    </a>
+  </li>
+</ul>
+
+The following tutorial implements a REST API server using the [Laravel](https://laravel.com/docs/10.x/readme) web development framework. The scenario is that of an e-commerce application. Database access in this application is managed through Laravel's Eloquent ORM.
+
+The source for this application can be found in the [Using ORMs with YugabyteDB](https://github.com/YugabyteDB-Samples/orm-examples/tree/master/php/laravel) repository.
+
+## Prerequisites
+
+This tutorial assumes that you have:
+
+- YugabyteDB up and running. Download and install YugabyteDB by following the steps in [Quick start](../../../../quick-start/).
+- PHP 7.3+
+- Laravel 8.40+
+- Composer (Depedency Manager for PHP)
+
+## Clone the "orm-examples" repository
+
+```sh
+$ git clone https://github.com/YugabyteDB-Samples/orm-examples.git
+```
+
+There are a number of options that can be customized in the `.env` file located in the Laravel project. Given YSQL's compatibility with the PostgreSQL language, the datasource `DB_CONNECTION` property is set to `pgsql`.
+
+## Build the application
+
+```sh
+$ cd orm-examples/php/laravel
```

Comment:
just changing directories, is there a build command

Reply:
Laravel apps get built with `php artisan serve` command.

Reply:
so what goes here